app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt

```diff
@@ -78,7 +77,8 @@ class OngoingListAdapter(
               (activity as Context).resources.getDimensionPixelSize(R.dimen.margin_28)
             }
           } else {
-            (activity as Context).resources.getDimensionPixelSize(R.dimen.margin_28)//this will be updated in next PR
+            // this will be updated in next PR
```

Comment:
Not entirely related to your PR but since you are changing this, can you create an issue for this if this is not fixed already. Or if it is fixed then we can remove this comment. Generally we shouldn't write such comments. If something needs to be done in the future we should create an issue for it and write it as `// TODO(issueNum):...`

Reply:
For this, I guess I had to confirm with the team like who has put this and take action accordingly.

Reply:
Yeah just check with whoever wrote this piece of code and add an issue instead of this comment (You can do this in another PR because its not related to linting at all)

Reply:
Discussed with Rajat and removed this line

Reply:
Resolving this on behalf of @vinitamurthi